docs/ee_promise_cache.js

```diff
@@ -8,8 +8,9 @@ export {
 // For testing only.
 export {clearPromiseCacheForTesting};
 
-// 250M objects in a FeatureCollection ought to be enough for anyone.
-const maxNumFeaturesExpected = 250000000;
+// 6M objects in a FeatureCollection: seems to be an EE bug that larger numbers
```

Comment:
Can you add that this was og 250000000 so reader has an idea what larger numbers means here

Reply:
Done.